mantle/kola/tests/misc/network.go

```diff
@@ -490,6 +546,20 @@ func checkExpectedMAC(c cluster.TestCluster, m platform.Machine, interfaceName,
 	}
 }
 
+func checkConnectionsUp(c cluster.TestCluster, m platform.Machine, connectionNames []string) error {
+	failedConnections := []string{}
+	for _, connection := range connectionNames {
+		connectionStatus := string(c.MustSSH(m, fmt.Sprintf("nmcli -f GENERAL.STATE connection show %s", connection)))
+		if !strings.HasSuffix(connectionStatus, "activated") {
+			failedConnections = append(failedConnections, connection)
```

Comment:
Should add the connection status that was found, so that it shows up in the error message for easier debugging. E.g. something like

```suggestion
			failedConnections = append(failedConnections, fmt.Sprintf("%s (state: %s)", connection, state))
```

Reply:
since I changed to use the '--active' flag then there is no status.. so I'll leave it at that, what do you say?